config/settings/base.py

```diff
@@ -228,3 +229,7 @@
 # Should always be False in production. Can be set to True in local environments
 # to serve static files even when DEBUG is False
 DJANGO_SERVE_STATIC = False
+
+CLIENT_BASE_URL = os.getenv("CLIENT_BASE_URL")
+CLIENT_KEY = os.getenv("CLIENT_KEY")
```

Comment:
I'm not sure these are the most descriptive names. Perhaps something like `ROSETTA_API_URL` and `ROSETTA_API_KEY` would be better?

Reply:
The wording "BASE" suggests that there may be more added to the URL path.

Same as below 
> The environment variable CLIENT_BASE_URL that uses Rosetta API can be changed to ROSETTA_CLIENT_BASE_URL to make it more clear and what `settings.CLIENT_BASE_URL` is using.

So the **environment** variables
 `CLIENT_BASE_URL` -> `ROSETTA_CLIENT_BASE_URL`,  
 `CLIENT_KEY` -> `ROSETTA_CLIENT_KEY`. 
Does this look good to change!